Structure_Engine/Compute/Integrate.cs

```diff
@@ -142,6 +178,9 @@ public static partial class Compute
         [MultiOutput(1, "constants", "The section constants calculated based on the provided section profile.")]
         public static Output<IProfile, Dictionary<string, double>> Integrate(IProfile profile, double tolerance = Tolerance.Distance)
         {
+            if (profile.IsNull())
+                return null;
```

Comment:
Not sure if we have a convention for what to return to multi outputs if one of the items is null, but would probably rather go with something like returning a correct output type, but setting both of the items in it to null. Just nitpicking, but wanting to keep it clean for UIs etc.

@FraserGreenroyd @adecler any ideas?

Reply:
@IsakNaslundBh I had to try to figure out something for this when I added null checks to DisplayMeshResults. The only solution I found which removed the UI errors relating to mismatched output types was to return a MultiOutput containing empty objects of the expected types - returning a MultiOutput containing nulls resulted in the same error as just returning null when I tried that.

I'm aware that it's probably not a good idea to do that for most methods though, since you wouldn't want the empty objects to be mistaken for real ones, so maybe a UI fix is needed to handle nulls in MultiOutputs. (Assuming there's no other way to fix it, maybe casting the nulls into the expected types?)

Reply:
```
            if (profile.IsNull())
                return new Output<IProfile, Dictionary<string, double>> { Item1 = null, Item2 = null };
```

I've gone for returning nulls - the file above is now outdated.

Reply:
Yeah that seems to work - it doesn't when I try to do the same thing for DisplayMeshResults. I'd guess it's because one of the return types is a list (a nested one, even.) Alright, well I guess you can disregard most of what I wrote above.